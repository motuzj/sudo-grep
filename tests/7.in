search one word + ignore case + invert match
--ignore-case --invert-match "n"
