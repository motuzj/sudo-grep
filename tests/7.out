Lorem ipsum DoloR sit amet,
Arcu vitae sed.
Ultrices vitae auctor eu augue ut lectus arcu.
Quis auctor elit sed vulputate mi sit amet.
aasfdasdfasdfsdaF
